build.rs

```diff
@@ -1,14 +1,16 @@
-use std::{collections::BTreeMap, env, sync::atomic::Ordering};
+use std::{collections::BTreeMap, env, iter, sync::atomic::Ordering};
 
 fn main() {
     println!("cargo:rerun-if-changed=build.rs");
+    configure_check_cfg();
 
     let target = env::var("TARGET").unwrap();
     let cwd = env::current_dir().unwrap();
 
     println!("cargo:compiler-rt={}", cwd.join("compiler-rt").display());
 
     // Activate libm's unstable features to make full use of Nightly.
+    println!("cargo::rustc-check-cfg=cfg(feature, values(\"unstable\"))");
     println!("cargo:rustc-cfg=feature=\"unstable\"");
```

Comment:
Should probably be replaced by a real custom config and not a faux Cargo feature, ie `unstable` and not `feature = "unstable"`.

Reply:
This comes from libm so I could probably do it here https://github.com/rust-lang/libm/pull/296

Thanks for the quick review